internal/services/network/network_watcher_flow_log_resource_test.go

```diff
@@ -285,7 +301,8 @@ resource "azurerm_storage_account" "test" {
 }
 
 func (r NetworkWatcherFlowLogResource) basicConfig(data acceptance.TestData) string {
-	return fmt.Sprintf(`
+	if !features.FivePointOhBeta() {
+		return fmt.Sprintf(`
```

Comment:
I see that the feature flag has been applied to all the test configs, which isn't entirely necessary. 

This doesn't need to be updated now but for future changes like this where a property is superseded, you can save some effort by updating all the test configs, except for one, to use the new property. 

One config should use the feature flag as you've done here to continue using the deprecated property, ensuring that it still works up until it is removed. But returning a config with the new property when running in 5.0 mode so we don't get any failing tests.

Reply:
Thank you. I followed the guide and assumed it had to be done everywhere. Maybe adding this piece of wisdom to the deprecation guide is a good idea. :)